tests/unittest/llmapi/apps/_test_openai_chat_json.py

```
@@ -0,0 +1,148 @@
+            "content":
+            f"Give an example JSON for an employee profile that "
+            f"fits this schema: {user_profile_schema}",
+        },
+    ]
+
+    first_json = _create_and_validate_response(messages)
+
+    messages.extend([
+        {
+            "role": "assistant",
+            "content": first_message.content,
```

Ruff (0.12.2)

Undefined name `first_message`

(F821)